Userland/Utilities/base64.cpp

```diff
@@ -14,10 +14,12 @@
 
 int main(int argc, char** argv)
 {
+#ifdef __serenity__
```

Comment:
I'm not sure I like those `#ifdef`s. Maybe we should add a new library for compatibility functions and just link all targets against that.

GCC supports `#include_next` so we can "patch" the host's `<unistd.h>` header to make `pledge()`, `unveil()`, etc. available.

Reply:
The `pledge()`/`unveil()` stuff isn't really portable to anything but SerenityOS, even for OpenBSD I seriously doubt that the pledges are actually compatible across the two operating systems.

NetBSD has a libcompat to deal with the portability of host tools. I'm not sure if we really should just stub out `pledge()`/`unveil()` and that could pose problems for OpenBSD too.